custom_components/ttlock/api.py

```diff
@@ -142,11 +143,25 @@ def lock_connectable(lock) -> bool:
             )
             return has_gateway or has_wifi
 
-        return [lock["lockId"] for lock in res["list"] if lock_connectable(lock)]
+        locks = [lock["lockId"] for lock in res_local["list"] if lock_connectable(lock)]
+        locks += [lock["lockId"] for lock in res_remote["list"] if lock_connectable(lock)]
+
+        return locks
 
     async def get_lock(self, lock_id: int) -> Lock:
         """Get a lock by ID."""
-        res = await self.get("lock/detail", lockId=lock_id)
+        try:
+            res = await self.get("lock/detail", lockId=lock_id)
```

Comment:
For users with a lot of locks there are already some problems with hitting the rate limits of the API.

Is there a way we could potentially track what kind of lock we're referencing so that features can make the right decision rather than having to try both APIs?